### DISCUSSION TITLE
What about the possibility of totalling columns?

### DISCUSSION THREAD

**Discussion by EmoCthulhu** [EddieDover/Theater-of-the-Mind#53] [Ideas]
Wondering if we could add keys to the value strings, like "total": true, and then have a special row at the end where you have blanks everywhere except where the columns are that should be totalled. It could then be "type": "total", and the module would then display text as Total: 3200 or whatever.

**Comment by EddieDover** [comment]
Not a bad idea but I'll have to think on how to implement it with the least amount of breakage while maximizing functionality.
